src/Session.php

```diff
@@ -67,7 +67,10 @@ public function isStarted()
      */
     public function start()
     {
-        $this->driver->start();
+        // start session if needed
```

Comment:
this comment is useless. The code is clear enough (and the phpdoc already says it too)